src/nvim/testdir/test_ruby.vim

```diff
@@ -4,44 +4,20 @@ if !has('ruby')
   finish
 end
 
+" Helper function as there is no builtin rubyeval() function similar
+" to perleval, luaevel() or pyeval().
+func RubyEval(ruby_expr)
```

Comment:
@alexgenco Can you port `rubyeval` from patch 8.1.1056, https://github.com/vim/vim/commit/e99be0e6d28fad96efd2b2be23fa38e7559e80e1? You can skip Vim's `+blob` feature and the tests from patch 8.1.0977 if needed. `+blob` patches haven't been ported yet.

Reply:
I'll take a crack at it